src/work-allocation/components/derived-icon-field/derived-icon-field.component.html

```diff
@@ -0,0 +1,5 @@
+<svg *ngIf="task&&sourceColumn&&task[sourceColumn]==matchValue" class="hmcts-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25" path="" style="fill: orange">
```

Comment:
Should be `===` and it would benefit from spaces either side of the `&&`s.

Alternatively - and this would be my preference - this could be a single getter in the component, which reduces the binding effort.